README.md

```
@@ -1,6 +1,465 @@
 # Tapaculo
-A lightweight Rust server that handles real-time and turn-based communication for any multiplayer client interactions
+
+A lightweight Rust server that handles real-time and turn-based communication for any multiplayer client interactions.
 
 <h1 align="center">
   <img width="300px" src="https://raw.githubusercontent.com/AlexTrebs/tapaculo/refs/heads/main/docs/images/icon.png" />
 </h1>
+
+## Features
+
+- ** Pluggable PubSub**: Swap between in-memory and Redis backends without code changes
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)